biscuit-auth/src/format/schema.proto

```diff
@@ -178,3 +181,39 @@ message ThirdPartyBlockContents {
   required bytes payload = 1;
   required ExternalSignature externalSignature = 2;
 }
+
+message AuthorizerSnapshot {
+  repeated string symbols = 1;
+  optional uint32 version = 2;
+  repeated SnapshotBlock blocks = 3;
+  required SnapshotBlock authorizerBlock = 4;
+  repeated GeneratedFacts generatedFacts = 5;
+  repeated Policy policies = 6;
```

Comment:
nitpick, but `policies` should go near `authorizerBlock` (and maybe named `authorizerPolicies` to make it clearer that they are linked)

Reply:
done in 927761c